service/instance.py

```diff
@@ -1608,7 +1699,8 @@ def _extra_openstack_args(core_identity, ex_metadata={}):
         # FIXME: In a new PR, allow user to select the keypair for launching
         user_key = user_keys[0]
         ex_keyname = user_key.name
-    return {"ex_metadata": ex_metadata, "ex_keyname": ex_keyname}
+    ex_security_groups = ["atmosphere"]
```

Comment:
We need a better way to tie this in than to do a hard-coded value.. how about something like:
```
security_group_name = core_identity.provider.get_config("network", "security_group_name", "atmosphere")
ex_security_groups = [security_group_name]
```
Similarly,[ in the area where we call user_security_group_init
](https://github.com/cyverse/atmosphere/pull/277/files#diff-545eec308259c382ab4919797a49db52R1342) we will want something similar:
```
def security_group_init(core_identity, max_attempts=3):
    has_secret = core_identity.get_credential('secret') is not None
+  security_group_name = core_identityprovider.get_config("network", "security_group_name", "atmosphere")
    if has_secret:
        return admin_security_group_init(core_identity)
    return user_security_group_init(core_identity, security_group_name=security_group_name)
```